src/Arcus.WebApi.Unit/Security/Authentication/CertificateAuthenticationAttributeTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.WebApi.Unit.Hosting;
+using Arcus.WebApi.Unit.Security.Doubles;
+using Xunit;
+using static Arcus.WebApi.Unit.Security.Authentication.CertificateAuthenticationOnMethodController;
+
+namespace Arcus.WebApi.Unit.Security.Authentication
```

Comment:
We should have a test for `Thumbprint` as well

Reply:
That's done I think.

Reply:
Thanks!